utils/zerovec/src/varzerovec/mod.rs

```diff
@@ -442,6 +433,13 @@ impl<'a, T: AsVarULE> VarZeroVec<'a, T> {
     pub fn get_serializable_bytes(elements: &[T]) -> Option<Vec<u8>> {
```

Comment:
Mostly for tests and stuff, because I want to avoid roundtripping through VZV. I can mark it as `doc(hidden)` to make it "public but not in the advertised API" though.